auth.js

```diff
@@ -133,23 +137,13 @@ let registerLocalUser = function(req,res){
         return util.apiResponse(req, res, 400, "Invalid username.");
     }
 
-    if(username in localUsers){
```

Comment:
This function is to check if user name is exist in database or not. you should not remove it.